astropy/io/ascii/ui.py

```diff
@@ -271,8 +280,18 @@ def _guess(table, read_kwargs, format, fast_reader):
     format : str
         Table format
     fast_reader : bool
+    fast_reader : bool or dict
         Whether to use the C engine, can also be a dict with options which
-        default to ``False`` (default= ``True``)
+        default to ``False`` (default= ``True``); parameters for options dict:
+
+        use_fast_converter: bool
+            enable faster but slightly imprecise floating point conversion method
+        parallel: bool or int
+            multiprocessing conversion using ``cpu_count()`` or ``'number'`` processes
+        exponent_style: str
+            Character to use for exponent or ``'Fortran'`` to auto-detect any
```

Comment:
In "modern" style that would probably be `ForTran` then, for "Formula Translator", but the official name has gone to title-caps as of `Fortran 90` (roughly going along with not-ALL-UPPERCASE SOURCE CODE becoming generally tolerated :grimacing: ).